console/console-init/ui/src/AppLayout.tsx

```diff
@@ -19,8 +19,8 @@ import {
 } from "components";
 import { AppRoutes } from "Routes";
 import { AppNavHeader } from "./AppNavHeader";
-import brandImg from "./assets/images/logo.svg";
-import avatarImg from "./img_avatar.svg";
+import brandImg from "logo.svg";
```

Comment:
What is the diffference between the two logo files:
./console/console-init/ui/src/assets/images/logo.svg 
./console/console-init/ui/src/logo.sv
The application has always used the one from the assets/images directory.  Which is where the about_logo.svg is as well.  If not using the assets image directory anymore should it be removed?